src/commands/apm_cmds/version.js

```diff
@@ -1,7 +1,31 @@
-const fs = require('fs')
+const { ensureWeb3 } = require('../../helpers/web3-fallback')
+const fs = require('fs-extra')
 const findUp = require('find-up')
 const semver = require('semver')
+const APM = require('@aragon/apm')
 
+const bumpVersionAndUpdateManifest = async ({ module, bump, network, apmOptions, cwd }) => {
+
+  const web3 = await ensureWeb3(network)
+  const apm = APM(web3, apmOptions)
+
+  let repo = { version: '0.0.0' }
+  try {
+    repo = await apm.getLatestVersion(module.appName)
+  } catch (e) {
+
```

Comment:
Notice that we swallow possible errors here. So far the only case I could imagine was that `getLatestVersion()` fails due to the very first deployment. This however shouldn't propagate an error because we know we can then just go ahead with an initial 1.0.0 version.

Let me know if this needs adjustment, e.g. cases where this API fails without being the very first publish.